graylog2-server/src/main/java/org/graylog2/events/ClusterEventPeriodical.java

```diff
@@ -86,16 +86,18 @@ static DBCollection prepareCollection(final MongoConnection mongoConnection) {
 
         DBCollection coll = db.getCollection(COLLECTION_NAME);
 
+        /* FIXME: Not supported by Fongo
         if (coll.isCapped()) {
             LOG.warn("The \"{}\" collection in MongoDB is capped which will cause problems. Please drop the collection.", COLLECTION_NAME);
         }
+        */
 
         coll.createIndex(DBSort
                 .asc("timestamp")
                 .asc("producer")
                 .asc("consumers"));
 
-        coll.setWriteConcern(WriteConcern.FSYNCED);
+        coll.setWriteConcern(WriteConcern.JOURNALED);
```

Comment:
Yes, I think replacing `FSYNCED` with `JOURNALED` is a good idea as it doesn't block a single server to fsync data to disk but makes sure that the document has been written to MongoDB's on disk journal.

* https://docs.mongodb.org/v3.2/reference/command/fsync/
* http://api.mongodb.org/java/3.2/com/mongodb/WriteConcern.html#FSYNCED
* http://api.mongodb.org/java/3.2/com/mongodb/WriteConcern.html#JOURNALED